src/content/reducers/profile-view.js

```diff
@@ -372,14 +370,16 @@ export type SelectorsForThread = {
   getFilteredThread: State => Thread,
   getRangeSelectionFilteredThread: State => Thread,
   getFuncStackInfo: State => FuncStackInfo,
-  getSelectedFuncStack: State => IndexIntoFuncStackTable,
-  getExpandedFuncStacks: State => IndexIntoFuncStackTable[],
+  getSelectedFuncStack: State => IndexIntoFuncStackTable | null,
+  getExpandedFuncStacks: State => Array<IndexIntoFuncStackTable | null>,
   getCallTree: State => ProfileTree.ProfileTreeClass,
   getFilteredThreadForFlameChart: State => Thread,
   getFuncStackInfoOfFilteredThreadForFlameChart: State => FuncStackInfo,
   getFuncStackMaxDepthForFlameChart: State => number,
   getStackTimingByDepthForFlameChart: State => StackTiming.StackTimingByDepth,
   getLeafCategoryStackTimingForFlameChart: State => StackTiming.StackTimingByDepth,
+  getFriendlyThreadName: State => string,
+  getThreadProcessDetails: State => string,
```

Comment:
It's out of scope of this patch a bit, but I was cross-comparing these lines with:

```javascript
    selectorsForThreads[threadIndex] = {
      getThread,
      getViewOptions,
      getCallTreeFilters,
      ...
      getStackTimingByDepthForFlameChart,
      getLeafCategoryStackTimingForFlameChart,
      getFriendlyThreadName,
      getThreadProcessDetails,
    };
```